sdk/postgresql/azure-resourcemanager-postgresql/src/test/java/com/azure/resourcemanager/postgresql/PostgreSqlManagerTests.java

```diff
@@ -0,0 +1,113 @@
+import com.azure.core.management.profile.AzureProfile;
+import com.azure.core.test.TestBase;
+import com.azure.core.test.annotation.DoNotRecord;
+import com.azure.core.util.Configuration;
+import com.azure.core.util.CoreUtils;
+import com.azure.identity.DefaultAzureCredentialBuilder;
+import com.azure.resourcemanager.postgresql.models.*;
+import com.azure.resourcemanager.resources.ResourceManager;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.Random;
+
+public class PostgreSqlManagerTests extends TestBase {
+    private static final Random RANDOM = new Random();
+    private static final Region REGION = Region.EUROPE_WEST;
+    private String resourceGroupName = "rg" + randomPadding();
+    private PostgreSqlManager postgreSqlManager;
+    private ResourceManager resourceManager;
+    private boolean testEnv;
+
+    @Override
+    public void beforeTest() {
+        final TokenCredential credential = new DefaultAzureCredentialBuilder().build();
+        final AzureProfile profile = new AzureProfile(AzureEnvironment.AZURE);
+
+        postgreSqlManager = PostgreSqlManager
+            .configure()
+            .withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC))
+            .authenticate(credential, profile);
+
+        resourceManager = ResourceManager
+            .configure()
+            .withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC))
+            .authenticate(credential, profile)
+            .withDefaultSubscription();
+
+        // use AZURE_RESOURCE_GROUP_NAME if run in LIVE CI
+        String testResourceGroup = Configuration.getGlobalConfiguration().get("AZURE_RESOURCE_GROUP_NAME");
+        testEnv = !CoreUtils.isNullOrEmpty(testResourceGroup);
+        if (testEnv) {
+            resourceGroupName = testResourceGroup;
+        } else {
+            resourceManager.resourceGroups()
+                .define(resourceGroupName)
+                .withRegion(REGION)
+                .create();
+        }
+    }
+
+    @Override
+    protected void afterTest() {
+        if (!testEnv) {
+            resourceManager.resourceGroups().beginDeleteByName(resourceGroupName);
+        }
+    }
+
+    @Test
+    @DoNotRecord(skipInPlayback = true)
+    public void testCreateServer() {
+        Server server = null;
+        String randomPadding = randomPadding();
+        try {
+            String serverName = "postgresql" + randomPadding;
+            String saUserName = "sqlsa" + randomPadding;
+            // @embedmeStart
+            server = postgreSqlManager.servers()
+                .define(serverName)
+                .withRegion(REGION)
+                .withExistingResourceGroup(resourceGroupName)
+                .withProperties(
+                    new ServerPropertiesForDefaultCreate()
+                        .withAdministratorLogin(saUserName)
+                        .withAdministratorLoginPassword("!QA2ws#ED4rf")
```

Comment:
same here for comment on password. we don't want user copy this and just use the password.

Reply:
The password setting has been changed to randomly generated.